src/ast/value.rs

```diff
@@ -26,10 +26,25 @@ use bigdecimal::BigDecimal;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
-use crate::ast::Ident;
+use crate::{ast::Ident, tokenizer::Span};
 #[cfg(feature = "visitor")]
 use sqlparser_derive::{Visit, VisitMut};
 
+/// Primitive SQL values such as number and string
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct ValueWrapper {
+    pub value: Value,
+    pub span: Span,
+}
```

Comment:
Do we want `Span` to influence `ValueWrapper`'s implementations of `PartialEq`, `PartialOrd`, and `Hash`?

Reply:
I don't know. What do you think is best?

Reply:
Based on previous feedback I got from @alamb, I think we should manually implement those traits on `ValueWrapper`, so that `span` is ignored

Reply:
Ok, I made the change